force-app/main/default/objects/CodeSet/fields/CodeSetType.field-meta.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>CodeSetType</fullName>
```

Comment:
@deepakmulamalla Missing field description.

Reply:
@cgiSunilM This is a standard HealthCloud field for CodeSet. We haven't done any customization around it we just committed the object to the repository.
@Kirannpatil97